src/app/functions/server/refreshStoredScanResults.ts

```diff
@@ -38,7 +38,7 @@ export async function refreshStoredScanResults(profileId: number) {
       }),
     );
 
-    // Record any new scans.
+    // Record any new scans, or change in existing scan status.
```

Comment:
Thanks for updating the comment! I was just typing a question about it while the [commit](https://github.com/mozilla/blurts-server/pull/3686/commits/2c8f4116889bf186c7917a9a19abb94f3fb44409) came in. ⚡